grails-app/services/org/pih/warehouse/glAccount/GlAccountService.groovy

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.glAccount
+
+import org.pih.warehouse.core.GlAccount
+
+
+class GlAccountService {
+
+    List<GlAccount> getGlAccounts (Map params) {
+        def max = params.max ? params.int("max") : null
+        def offset = params.offset ? params.int("offset") : null
+
+        return GlAccount.createCriteria().list(max: max, offset: offset) {
+            if (params.active != null) {
```

Comment:
I'm wondering if there is a need to do `!= null`?
Isn't 
```suggestion
            if (params.active) {
```
enough?

Reply:
nevermind, I think I get it now - if we had `false` we wouldn't "go" into this `if`